src/tests/ftest/util/job_manager_utils.py

```diff
@@ -304,6 +358,15 @@ def assign_environment_default(self, env_vars):
         """
         self.export.update_default(env_vars.get_list())
 
+    def get_lib_path(self):
```

Comment:
Should this be moved to the common `JobManager` class? Or maybe a `class MpiJobManager(JobManager):` class that then `Mpirun` and `Orterun` inherit?